### DISCUSSION TITLE
Controlling layout of diagrams

### DISCUSSION THREAD

**Discussion by Vaneeza-7** [plantuml/plantuml#1983] [Q&A]
Is there any way to control the layout of diagrams in plantUML? For example, limiting the number of lines crossing each other, etc. Algorithms for controlling layout like this exist on Graphviz library but is there anything similar in plantUML as well?

Currently, I'm especially struggling with entity relationship diagrams. They're just so messy and not even understandable.

**Comment by Evan0000000000** [comment]
the position of the designers seems to be that you shouldn't care about that kind of thing, but I don't understand it myself.
